### ISSUE TITLE
Support for Bundle in Bundle Encapsulation (BIBE)

### ISSUE THREAD

**Issue by amontilla-spatiam** [nasa/HDTN#63]
Hi there!
We would like to request the support for Bundle in Bundle Encapsulation (BIBE) according to the IETF DTN WG draft: https://datatracker.ietf.org/doc/html/draft-ietf-dtn-bibect

This is important to integrate HDTN into the SPATIAM DTN Platform, and the target LunaNet Service Provider architecture, i.e. having HDTN as a gateway node.

As reference, there is the ION implementation available to test again, and an initial implementation on µD3TN as well.

Regards and big thanks;
Alberto Sr. 
Spatiam Corporation

**Comment by scburleigh** [comment]
I agree, BIBE is going to be a very important BP capability going forward.

**Comment by ccaini** [comment]
I agree with Alberto and Scott. BIBE is going to be added to Unibo-BP too in the next months (a thesis has just started). The larger the support for BIBE, the better.
Carlo Caini
University of Bologna
